options/locale/locale_en-US.ini

```diff
@@ -277,6 +277,9 @@ org_no_results = No matching organizations found.
 code_no_results = No source code matching your search term found.
 code_search_results = Search results for '%s'
 code_last_indexed_at = Last indexed %s
+relevant_repositories_tooltip = Repositories that are forks or don't have a topic and description aren't being shown.
```

Comment:
Not sure if the repeated `no` is good practice of the English language.

Reply:
1. The original string does not mention the icons also being relevant
2. There is uncertainty on how to interpret the string otherwise: `that don't have a topic, icon, and description` -> does that mean (only) repositories with icon and description but no topic will not be shown?

That's at least my reasoning for the additional `no`s.